### PR TITLE
[8.0][website_mail_bg_color] Color picker for mail snippets

### PR DESCRIPTION
**Pull request by yajo**
This module extends the functionality of the website mail designer to support setting a custom background color to any snippet and allow you to customize it.

It would not be hard to extend to all website snippets, but I prefer to keep that for a future version.

@rafaelbn

### CODE REVIEW

File: website_mail_snippet_bg_color/README.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Email Snippets Background Color Chooser
```
Comment (elicoidal) [inline website_mail_snippet_bg_color/README.rst]: s/Chooser/Picker?

File: website_mail_snippet_bg_color/README.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+=======================================
+
+This module extends the functionality of the website mail designer to support
+setting a custom background color to any snippet and allow you to customize it.
```
Comment (elicoidal) [inline website_mail_snippet_bg_color/README.rst]: s/allow/allowing

File: website_mail_snippet_bg_color/README.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+#. Install any module that makes use of the website mail designer, such as
+   ``mass_mailing``.
+#. Edit an email with the website mail designer.
+#. Drag & drop any snippet into the mail body.
+#. Choose *Customize > Choose Background Color*.
```
Comment (elicoidal) [inline website_mail_snippet_bg_color/README.rst]: Click on _Customize > Choose Background Color_.

File: website_mail_snippet_bg_color/README.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+#. Pick any color, or enter its HTML code in the box, or press *Clear* to
```
Comment (elicoidal) [inline website_mail_snippet_bg_color/README.rst]: Pick any color or enter its HTML code in the box. Press _Clear_ to
   remove it.

File: website_mail_snippet_bg_color/views/snippets.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <div
+            data-snippet-option-id='bg_color_chooser'
+            data-selector="[data-oe-field='body_html'] > div,
+                           .oe_snippet_body,
+                           .bg_color_chooser"
+            data-selector-siblings="[data-oe-field='body_html'] > *"
+            data-selector-children="[data-oe-field='body_html']">
+            <li>
+                <a href="#" class="button js_bg_color_chooser">
+                    Choose Background Color
```
Comment (elicoidal) [inline website_mail_snippet_bg_color/views/snippets.xml]: s/Choose/Select or Pick?

**Comment by elicoidal** [comment]
Thanks for the contribution!
LGTM

**Comment by dgzurita** [comment]
Tested in runbot :+1:

**Comment by pedrobaeza** [comment]
Tested also :+1:

**Comment by rafaelbn** [comment]
:+1: tested un local

**Pull request merged**